### DISCUSSION TITLE
FCITX5 doesn't work QT apps under X11 on Arch Plasma

### DISCUSSION THREAD

**Discussion by HubKing** [fcitx/fcitx5#1203] [Q&A] [answered]
It works for both GTK and QT apps in the Wayland session. But if I log into the X11 session, it only works on GTK apps like FireFox, but not on QT apps like KWrite or the System Settings. How to fix?

fctix5-qt6 package has been installed.

**Comment by wengxt** [comment, marked as answer]
Under X11 you'd better have QT_IM_MODULE=fcitx if you have unset it per wayland self diagnose notification.

You can run fcitx5-diagnose to verify.

**Reply by HubKing** [reply]
Adding `export QT_IM_MODULE=fcitx` to `~/.xprofile` and rebooting the PC solved it.

**Reply by VixiKitsune** [reply]
> Adding `export QT_IM_MODULE=fcitx` to `~/.xprofile` and rebooting the PC solved it.

Hi, I'm having the same issue and I'm wondering where do I find ~/.xprofile?

**Reply by wengxt** [reply]
@VixiKitsune create your own one.
